source/grid/grid_in.cc

```diff
@@ -1473,8 +1952,8 @@ GridIn<dim, spacedim>::read_msh(std::istream &in)
   else
     AssertThrow(false, ExcInvalidGMSHInput(line));
 
-  // if file format is 2.0 or greater then we also have to read the rest of the
-  // header
+  // if file format is 2.0 or greater then we also have to read the rest of
+  // the header
```

Comment:
I have to admit that I don't know what happened here, but the remainder of the changes in this file are just wrapped around comments.